flytekit/clis/sdk_in_container/backfill.py

```diff
@@ -0,0 +1,174 @@
+import typing
+from datetime import datetime, timedelta
+
+import click
+
+from flytekit.clis.sdk_in_container.helpers import get_and_save_remote_with_click_context
+from flytekit.clis.sdk_in_container.run import DateTimeType, DurationParamType
+
+_backfill_help = """
+Backfill command generates, registers a new workflow based on the input launchplan, that can be used to run an
+automated backfill. The workflow can be managed using the Flyte UI and can be canceled, relaunched, recovered and
+is implicitly cached. 
+"""
+
+
+def resolve_backfill_window(
+    from_date: datetime = None,
+    to_date: datetime = None,
+    window: timedelta = None,
+) -> typing.Tuple[datetime, datetime]:
+    """
+    Resolves the from_date -> to_date
+    """
+    if from_date and to_date and window:
+        raise click.BadParameter("Cannot use from-date, to-date and duration. Use any two")
```

Comment:
do you mean `window` instead of `duration`?

Reply:
yes, will update